external-providers/dotnet-external-provider/pkg/dotnet/filepath_filtering_test.go

```diff
@@ -0,0 +1,304 @@
+package dotnet
+
+import (
+	"runtime"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/konveyor/analyzer-lsp/provider"
+	"go.lsp.dev/protocol"
+	"go.lsp.dev/uri"
+)
+
+func TestNormalizePathForComparison(t *testing.T) {
```

Comment:
We should move the test next to the lib.go

Reply:
5fbc1c9